packages/form-js-viewer/src/Form.js

```diff
@@ -130,12 +130,31 @@ export default class Form {
    * @returns { { data: Data, errors: Errors } }
    */
   submit() {
-    this.validate();
+    const { properties = {} } = this._getState();
 
-    const {
-      data,
-      errors
-    } = this._getState();
+    const { readOnly } = properties;

```

Comment:
We could actually argue that if `properties.readOnly` is set an error should be thrown. 

How can a user submit a read-only marked form? What would be the use-case?

Reply:
No use case. My idea was that disabled form fields are missing in `data` and that submitting a disabled form would mean that all form fields would be missing in `data`. We could, on the other hand, throw an error, yes.

Reply:
I'd rather throw an error. Submitting a programmatically disabled form is a miss-use of the API that we should indicate.

Reply:
Okay, agreed.

Reply:
Done (cf. https://github.com/bpmn-io/form-js/pull/85/files#diff-53d9b079c7db646b79736d7ab07cd7d3108049b780d14637f41c188a4f6ffc03R138).